doc/advanced.rst

```diff
@@ -680,6 +680,16 @@ method::
         // ...
     }
 
+.. caution::
+
+    Globals are fetched once from extensions and then cached for the lifetime
+    of the Twig environment. It means that globals should not be used to store
+    values that can change during the lifetime of the Twig environment. For
+    instance, if you're using an application server like RoadRunner of
```

Comment:
`RoadRunner of FrakenPHP`

->

`RoadRunner or FrankenPHP`

Reply:
Fixed